### ISSUE TITLE
Feature Request: Clone Board without cards

### ISSUE THREAD

**Issue by xet7** [wekan/wekan#4726]
Click `Clone Board`. Then there would be popup:
```
[_] Without cards

[Clone]  [Cancel]
```
It could be implemented like this:

1. If `[X] Without cards` is checked
2. In `Clone Board` code, do not run that part of code that copies cards.

Maybe some other options later.

Related discussion: https://github.com/wekan/wekan/discussions/4724

**Comment by e-gaulue** [comment]
In my case, I'd like to clone boards without cloning archived cards.

**Comment by xet7** [comment]
@e-gaulue 

It looks like you would like to have some features. So:

a) Are you planning to send some pull requests that implement those features?

b) Do you have some developers that could send those pull requests?

c) Are you planning to fund development of those features at https://wekan.team/commercial-support/ ?

**Comment by e-gaulue** [comment]
@xet7

According to my experience, when I really need something I develop it for me. Making it pullable is about 5 to 10 times longer   because constraints are not the same (I think you really know about it). But I'm always happy to share my recipes.

Thinking about it since I wrote my previous post, I will do it directly in Mongo, with something like "list boards, if board is duplicated then look for archived cards and remove them". Of course, a lovely button in the UI to accomplish this could be great, but I haven't got those skills. 

They are so much things you can do in wekan just through mongo and recipes. Wekan wiki could receive/organise those recipes.

We have worked for 3 months in 2020 with an internship on wekan, for no results. We tried to use it as a communication datastore 1 communication = 1 card, with 70 k boards and 70 Go of attachments. Not so much, but we reached wekan limits you are working on (according to your page about DOS). Finally, we have develop a typical old fashion application (PHP/MariaDB). It's not so reactive and color-full, but it makes the job.

**Comment by xet7** [comment]
@e-gaulue 

> Not so much, but we reached wekan limits you are working on (according to your page about DOS).

Yes, I'm fixing those limits, with some internal code changes (like less code). UI and features stays mostly the same.

While you can help with those migrations etc if you have time, I will anyway fix them, release newest automatic upgrade to WeKan Snap Stable, etc.
